packages/core/i18n/src/Translation.ts

```diff
@@ -43,7 +43,7 @@ export const i18nTComponent = <MessageSource extends Record<string, any>>(i18n:
 
     return (): VNodeChild => {
       const keys = Object.keys(slots).filter(key => key !== '_')
-      const sourceString = (i18n || props.i18n).messages[(props.keypath as string)].toString()
+      const sourceString = (i18n || props.i18n).messages[(props.keypath as string)]?.toString() || ''
```

Comment:
is this trying to account for an empty keypath string..?

Reply:
Some how it looks like a DD alert was fired where `i18n.messages[keypath]` is undefined